math_hurdler.py

```diff
@@ -313,11 +311,10 @@ def evaluate_answer(answer):
                 self.set_gameover(True)
                 if self.last_answer_index >= 0:
                     self.buttons[self.last_answer_index].set_color(
-                        Color.RED, False)
+                        Color.RED)
 
             self.buttons[self.question.answer_index].set_color(
-                Color.GREEN, False)
-        
+                Color.GREEN)
 
         while self.running:
             # Processing Gtk Events
```

Comment:
I know this wasn't your change but could you please remove redundant comments like this?

Reply:
Yes yes, sure